src/type_checkers/where_statement.cr

```diff
@@ -1,5 +1,26 @@
 module Mint
   class TypeChecker
+    def static_type_signature_for_inline_function(node : Ast::WhereStatement) : Checkable
+      expression =
+        node.expression
+
+      if expression.is_a?(Ast::InlineFunction)
```

Comment:
The code for calculating  the static type signature of the `Ast::InlineFunction` should be in its type-checking file, because we might use it later.

Reply:
I moved it to `inline_function.cr`